resources/references/extension-reference/twig-function-reference.md

```diff
@@ -0,0 +1,35 @@
+# Shopware's twig functions
+
+In Shopware, we extend Twig's functionality by custom ones. See our own actions below.
+
+## Functions
+
+| Function | Description | --- |
+| --- | --- | --- | 
+| `sw_extends` | Inherits from another file with support for multi inheritance. The API is the same like in twigs default `extends` | --- | 
+| `sw_include` | Includes template partials with support for multi inheritance. The API is the same like in twigs default `include`
+| --- | 
+| `sw_icon` | Displays an icon from a given icon set| See [Add thumbnail](./../../../guides/plugins/plugins/storefront/use-media-thumbnails.md) guide for details.  | 
+| `sw_thumbnails` | Renders a <img> tag with correctly configured “srcset” and “sizes” attributes based on the provided parameters | See [Add custom icon](./../../../guides/plugins/plugins/storefront/add-icons.md) guide for more information. | 
```

Comment:
You mixed up the links here, didn't you?

Reply:
Seems like that, it should be swapped with the one above.